src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design/build/netcoreapp1.0/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.targets

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PrecompileContentRoot Condition="'$(PrecompileContentRoot)'==''">$(MSBuildProjectDirectory)</PrecompileContentRoot>
```

Comment:
Need names \ a prefix since this is a property a user could override from the proj file. `MvcRazorContentRoot`?